Source/Core/Core/PowerPC/Jit64/RegCache/GPRRegCache.h

```diff
@@ -11,11 +11,20 @@ class GPRRegCache final : public RegCache
 {
 public:
   explicit GPRRegCache(Jit64& jit);
+
+  using RegCache::IsImm;
```

Comment:
Is there a reason this is needed here but not in `FPURegCache.h`?

Reply:
Removed.